Applications/DataExplorer/NetCdfDialog/NetCdfConfigureDialog.cpp

```diff
@@ -216,81 +200,55 @@ void NetCdfConfigureDialog::setDimensionSelect()
         spinBoxDim4->setValue(0);
 }
 
-void NetCdfConfigureDialog::getDimEdges(int dimId, unsigned &size, double &firstValue, double &lastValue)
+void NetCdfConfigureDialog::getDimEdges(std::string const& name, unsigned& size,
+                                        double& firstValue, double& lastValue)
```

Comment:
better to return a tuple.

Reply:
I'm returning three values here (size, first, last) ... putting these into one tuple seems strange. Also it'd require a ton of changes because the method is called quite often. Do you insist on this?

Reply:
no, it's a legacy code, so we keep it as is, until somebody needs to change it. Would be good opportunity to improve it now, though.